Documentation/getting-started-guide.md

```diff
@@ -9,19 +9,32 @@ If you're not on Linux, you should do all of this inside [the rkt Vagrant](https
 package main
 
 import (
-	"log"
-	"net/http"
+        "github.com/coreos/go-systemd/daemon"
+        "log"
+        "net"
+        "net/http"
 )
 
 func main() {
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		log.Printf("request from %v\n", r.RemoteAddr)
 		w.Write([]byte("hello\n"))
 	})
-	log.Fatal(http.ListenAndServe(":5000", nil))
+	ln, err := net.Listen("tcp", ":5000")
+	if err != nil {
+	fmt.Fprintf(os.Stderr, "%v\n", err)
+	os.Exit(1)
+	}
+	err = daemon.SdNotify("READY=1")
+	if err != nil {
+		log.Printf("daemon.SdNotify READY=1 return: %s", err)
+	}
+	log.Fatal(http.Serve(ln, nil))
 }
 ```
 
+To know more about rkt notification to systemd read [using rkt with systemd](using-rkt-with-systemd.md)
```

Comment:
I think we should have a separate small chapter in this document, and a separate dedicated example for a "systemd notification hello go example". Also we should add a disclaimer in this new chapter that this applies only to stage1 images based on systemd (there might be other implementations). Changing the default "hello go" example suggests systemd is a prerequisite.